diff --git a/tests/upstream/global/utils/adjustLayoutByDelta.test.ts b/tests/upstream/global/utils/adjustLayoutByDelta.test.ts
--- a/tests/upstream/global/utils/adjustLayoutByDelta.test.ts
+++ b/tests/upstream/global/utils/adjustLayoutByDelta.test.ts
@@ -1,7 +1,7 @@
 import { describe, expect, test } from "vitest";
-import type { Layout } from "../../components/group/types";
-import type { PanelConstraints } from "../../components/panel/types";
-import { adjustLayoutByDelta as adjustLayoutByDeltaExternal } from "./adjustLayoutByDelta";
+import type { Layout } from "../../../../src/components/group/types";
+import type { PanelConstraints } from "../../../../src/components/panel/types";
+import { adjustLayoutByDelta as adjustLayoutByDeltaExternal } from "../../../../src/global/utils/adjustLayoutByDelta";
 
 type Args = Parameters<typeof adjustLayoutByDeltaExternal>[0];
 
@@ -1078,12 +1078,7 @@ describe("adjustLayoutByDelta", () => {
       adjustLayoutByDelta({
         delta: 100,
         initialLayout: l([25, 25, 25, 25]),
-        panelConstraints: c([
-          {},
-          { minSize: 10 },
-          { minSize: 10 },
-          { minSize: 10 }
-        ]),
+        panelConstraints: c([{}, { minSize: 10 }, { minSize: 10 }, { minSize: 10 }]),
         prevLayout: l([25, 25, 25, 25])
       })
     ).toEqual(l([70, 10, 10, 10]));
@@ -1791,12 +1786,7 @@ describe("adjustLayoutByDelta", () => {
       adjustLayoutByDelta({
         delta: -50,
         initialLayout: l([25, 25, 25, 25]),
-        panelConstraints: c([
-          { minSize: 10 },
-          { minSize: 10 },
-          { minSize: 10 },
-          {}
-        ]),
+        panelConstraints: c([{ minSize: 10 }, { minSize: 10 }, { minSize: 10 }, {}]),
         pivotIndices: [2, 3],
         prevLayout: l([25, 25, 25, 25])
       })
